src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx

```diff
@@ -37,4 +37,4 @@ const CalloutCanvasTemplateStep: FC<CalloutCanvasTemplateStepProps> = ({ callout
     />
   );
 };
-export default CalloutCanvasTemplateStep;
+export default CalloutCanvasTemplate;
```

Comment:
Why is it renamed. It's not used anymore.

Reply:
It is going to be used when Callout templates are introduced. It will be used not as a step but as part of the Callout creation dialog. My logic was either delete it or prepare for future use.